### DISCUSSION TITLE
打开8760端口的swagger文档地址特别慢

### DISCUSSION THREAD

**Discussion by zqb666** [zuihou/zuihou-admin-cloud#218] [02.提问]
访问地址 http://ip:8760/api/doc.html#/home 
访问 swagger-resources 接口 特别慢
<img width="1790" alt="image" src="https://user-images.githubusercontent.com/6092525/200988120-d3abc8d0-d175-4dea-aaa8-e7ca3709f1dc.png">

**Comment by zuihou** [comment]
问题不大，自行克服
